snap/snapcraft.yaml

```diff
@@ -95,6 +95,8 @@ parts:
       - cargo
     stage-packages:
       - aria2
+    stage-snaps:
+      - ffmpeg/latest/edge
```

Comment:
even in edge ffmpeg is v4.x. I looked at yt-dlp and it turns out there are patched ffmpeg builds from yt-dlp developers that fix some bugs. And according docs, there were additional patches before v5.1 that I believe can affect TC https://github.com/yt-dlp/FFmpeg-Builds#historical-patches
So I guess we better use manual builds until ffmpeg snap gets updated